src/videojs-contrib-hls.js

```diff
@@ -587,6 +587,10 @@ class HlsHandler extends Component {
       this.tech_.trigger('progress');
     });
 
+    this.on(this.masterPlaylistController_, 'bandwidthupdate', function() {
+      this.tech_.trigger('bandwidthupdate');
```

Comment:
any reason to not trigger straight from the tech within MPC if the HLS source handler isn't doing anything other than forwarding the event to the tech?